plugins/SimulationView/SimulationViewMenuComponent.qml

```diff
@@ -133,7 +132,54 @@ Cura.ExpandableComponent
             width: parent.width
             model: layerViewTypes
             visible: !UM.SimulationView.compatibilityMode
-            style: UM.Theme.styles.combobox
+
+            background: Rectangle
+            {
+                implicitHeight: UM.Theme.getSize("setting_control").height;
+                implicitWidth: UM.Theme.getSize("setting_control").width;
+
+                color: ladyerTypeCombobox.hovered ? UM.Theme.getColor("setting_control_highlight") : UM.Theme.getColor("setting_control")
+                Behavior on color { ColorAnimation { duration: 50; } }
+
+                border.width: UM.Theme.getSize("default_lining").width;
+                border.color: ladyerTypeCombobox.hovered ? UM.Theme.getColor("setting_control_border_highlight") : UM.Theme.getColor("setting_control_border");
+                radius: UM.Theme.getSize("setting_control_radius").width
+            }
+
+            contentItem: Item
+            {
+                Label
```

Comment:
Use UM.Label (that way you can set way less default properties, so less duplication :))

Reply:
https://github.com/Ultimaker/Cura/pull/11336/commits/200e4df0254a5a3f4907331da6ee82a3030ba575